foundations/javascript_basics/fundamentals-1.md

```diff
@@ -12,12 +12,12 @@ This section contains a general overview of topics that you will learn in this l
 * What are operators, operands, and operations?
 * What is concatenation and what happens when you add numbers and strings together?
 * What are the different types of operators in JavaScript?
-* What is the difference between `==` and `===`?
+* What is the difference between loose equality(==) and strict equality(===)?
 * What are operator precedence values?
-* What are the increment/decrement operators?
-* What is the difference between prefixing and post-fixing them?
-* What are assignment operators?
-* What is the "Unary +" Operator?
+* What are the increment(++)/decrement(--) operators?
+* What is the difference between prefixing and postfixing them?
+* What are assignment(=) operators?
```

Comment:
I think a part of being able to answer this question confidently is knowing that the assignment operator is `=`. I don't like giving it away in the question.

Reply:
Makes sense. I wanted the questions to be consistent throughout, hence the brackets with the operators in them. I agree with the point you're making. I'll update the assignment operator question.